Makefile

```diff
@@ -28,6 +28,7 @@ GOTAGSLIST          := sqlite_unlock_notify sqlite_omit_load_extension
 
 # e.g. make GOTAGSCUSTOM=msgtrace
 GOTAGSLIST += ${GOTAGSCUSTOM}
+export GOTESTCOMMAND=gotestsum --format pkgname --jsonfile testresults.json --
```

Comment:
make it a conditional: if `gotestsum` presents, then use it. Otherwise, fallback to the go test.